lib/pbench/server/indexing_tarballs.py

```diff
@@ -275,37 +277,37 @@ def sigquit_handler(*args):
                                     # Turn off the SIGINT handler when not indexing.
                                     signal.signal(signal.SIGINT, signal.SIG_IGN)
                         except UnsupportedTarballFormat as e:
-                            idxctx.logger.warning("Unsupported tar ball format: {}", e)
-                            tb_res = 4
+                            idxctx.logger.warning(
+                                "{}: {}", error_code["TB_META_ABSENT"].message, e
+                            )
+                            tb_res = error_code["TB_META_ABSENT"]
                         except BadDate as e:
-                            idxctx.logger.warning("Bad Date: {!r}", e)
-                            tb_res = 5
+                            idxctx.logger.warning(
+                                "{}: {!r}", error_code["BAD_DATE"].message, e
+                            )
+                            tb_res = error_code["BAD_DATE"]
                         except FileNotFoundError as e:
-                            idxctx.logger.warning("No such file: {}", e)
-                            tb_res = 6
-                        except BadMDLogFormat as e:
                             idxctx.logger.warning(
-                                "The metadata.log file is curdled in" " tar ball: {}", e
```

Comment:
The new error message is much better, although we apparently no longer identify the tarball with the problem, which isn't good.

Reply:
I think it is a Github issue that it isn't aligning the right message with that particular exception.

Reply:
Yes, apparently the tarball name in the output comes from the exception...